appveyor.yml

```diff
@@ -0,0 +1,33 @@
+#---------------------------------#
+#      general configuration      #
+#---------------------------------#
+
+# version format
+version: 1.0.0.{build}
+
+# Do not build on tags (GitHub and BitBucket)
+skip_tags: true
+
+# Maximum number of concurrent jobs for the project
+max_jobs: 1
+
+#---------------------------------#
+#    environment configuration    #
+#---------------------------------#
+
+# Build worker image (VM template)
+image: Visual Studio 2017
```

Comment:
🤔  Is it worth trying VS2019 image?

Reply:
I would not yet want to remove VS2017, since it is still the "official" version used for GE.
How about configuring multiple images, so we test VS2017 _and_ VS2019?

Reply:
I don't think it will make a difference, but sure.
I only have VS2019 installed anyway 😊